### DISCUSSION TITLE
How to make standard equal indents in EPUB book?

### DISCUSSION THREAD

**Discussion by Entony777** [sumatrapdfreader/sumatrapdf#2910] [General]
I see different indents in different books.
Number 1 - very large, Number 2 - very small, especially on top and bottom, Number 3 - good equivalent indents.

![2222](https://user-images.githubusercontent.com/112500035/187480849-2bfcc94c-cb84-4c36-8ae2-26366cdbfed5.jpg)

How to make standard equal indents in all EPUB book?

Thanks

**Comment by GitHubRulesOK** [comment]
Analyse the indent you like is it several space bar characters? then edit the ebooks that use different methods and add the same indents into their text.

This one says its style is indented and the paragraph above is inset and the first one has none its "EB02BodyTextFullOut"

![image](https://user-images.githubusercontent.com/12754913/187512422-c4cfaa29-90c7-4e1d-bc7a-3cf67801efd9.png)
![image](https://user-images.githubusercontent.com/12754913/187512853-855f9547-b35c-4216-abc5-41edb52895c8.png)
